Mathlib/Analysis/Normed/Group/Bounded.lean

```diff
@@ -139,6 +139,12 @@ theorem Filter.Tendsto.op_one_isBoundedUnder_le {f : α → E} {g : α → F} {l
     (h_op : ∀ x y, ‖op x y‖ ≤ ‖x‖ * ‖y‖) : Tendsto (fun x => op (f x) (g x)) l (𝓝 1) :=
   hf.op_one_isBoundedUnder_le' hg op ⟨1, fun x y => (one_mul ‖x‖).symm ▸ h_op x y⟩
 
+lemma tendsto_norm_comp_cofinite_atTop_of_isClosedEmbedding {X E : Type*} [TopologicalSpace X]
```

Comment:
Can't you use some variable already in scope?

Reply:
Thank yes, I've made a multiplicative version now and added the to_additive flag